datahub-web-react/src/app/entity/dataJob/preview/Preview.tsx

```diff
@@ -68,6 +70,7 @@ export const Preview = ({
             urn={urn}
             description={description || ''}
             type="Data Task"
+            type={subType}
```

Comment:
This looks like we're overwriting the `type` prop, think this should be something like `subtype || 'Data Task'`

Reply:
thanks for catching this!